### PR TITLE
[INFINITY-1787]Handle arg-provided requirements.txt to run-tests.py correctly.

### PR DESCRIPTION
**Pull request by jsrodman**
Confusion ensued from "requirements_txt" not being text, but a filename.  Handled partially but not fully.

### CODE REVIEW

**Comment by mgummelt** [comment]
Please prefix your PRs with [JIRA-###]

File: tools/piputil.py [hunk 1]
```diff
@@ -59,16 +59,16 @@ def pip_install_dir(path, requirements_filepath):
 
     subprocess.check_call(pip_cmd)
 
-def create_requirementsfile(filename, req_txt=None):
-    if not req_txt:
-        req_txt = ('''
+def create_requirementsfile(filename, req_text=None):
+    if not req_text:
```
Comment (mgummelt) [inline tools/piputil.py]: `if req_text is None`

**Comment by jsrodman** [comment]
I always prefix my PRs with the INFINITY tag.

**Comment by jsrodman** [comment]
The part I know how to test has already passed, merging.

File: tools/piputil.py (hunk 1 above)
Comment (jsrodman) [inline tools/piputil.py]: An empty requirements.txt file is not valid.

Comment (mgummelt) [inline tools/piputil.py]: We should probably fail fast on invalid input, rather that silently converting it, but not a big deal.

**Pull request merged**